generator/road_access_generator.hpp

```diff
@@ -2,32 +2,45 @@
 
 #include "generator/collector_interface.hpp"
 #include "generator/feature_builder.hpp"
-#include "generator/intermediate_elements.hpp"
 #include "generator/osm_element.hpp"
+#include "generator/way_nodes_mapper.hpp"
 
 #include "routing/road_access.hpp"
 #include "routing/vehicle_mask.hpp"
 
 #include <array>
-#include <cstdint>
 #include <map>
 #include <memory>
 #include <optional>
-#include <ostream>
 #include <set>
 #include <string>
 #include <unordered_map>
 #include <vector>
 
+namespace routing
+{
+class OsmWay2FeaturePoint;
+
+template <class Sink> void Save(Sink & sink, RoadAccess::Type const & ac)
+{
+  WriteToSink(sink, static_cast<uint8_t>(ac));
+}
+
+template <class Source> void Load(Source & src, RoadAccess::Type & ac)
```

Comment:
```suggestion
template <class Source> RoadAccess::Type Load(Source & src)
```

Reply:
Function's overloading is used here with common notation: void Load(Source &, T &);